funnel/templates/macros.html.jinja2

```diff
@@ -706,3 +706,54 @@
+          {% else %}
+            <p  class="mui--text-light mui--text-body2 zero-bottom-margin">{% trans date=project.cfp_end_at_localized|datetime(format='dd MMM YYYY, hh:mm a') %}Accepting submissions till {{ date }}.{% endtrans %}<a href="{{ project.url_for('cfp') }}" class="mui--text-light mui--text-body2"> {% trans %}Click to edit the end date{% endtrans %}{{ faicon(icon='pencil', icon_size='caption', baseline=false, css_class='mui--text-light margin-left') }}</a></p>
+          {% endif %}
+        {%- endif %}
+      </div>
+      <div class="js-cfp-status {% if project.cfp_state.OPEN -%} mui--hide {%- endif %}">
+        <p class="mui--text-subhead mui--text-bold margin-bottom" data-cy="cfp-state">{% trans %}Submission status: OFF{% endtrans %}</p>
+        <p class="mui--text-light mui--text-body2 zero-bottom-margin">{% trans %}Not accepting submissions{% endtrans %}</p>
+      </div>
+    </div>
+    {%- if project.view_for('cfp').is_available() %}
+      <div class="flex-wrapper flex-wrapper--center">
+        <form action="{{ project.url_for('cfp_transition') }}" method="post" class="mui--d-inlineblock mui--align-top right-padding">
+          {{ csrf_tag() }}
+          <input type="checkbox" name="open" id="open-sub" class="switch-input js-toggle" {%- if project.cfp_state.OPEN  %}checked{%- endif %}/>
+          <label class="switch-label mui--pull-right" for="open-sub"></label>
+        </form>
+        <span data-toggle="tooltip" data-placement="top" title="{% trans %}Toggle to enable/disable submissions{% endtrans %}" aria-label="{% trans %}Open to receive submissions{% endtrans %}">{{ faicon(icon='info-circle', baseline=true) }}</span>
+      </div>
+    {%- endif %}
+  </div>
+{%- endmacro %}
+
+
+{%- macro add_submission_btn(project) %}
+  <div class="project-details__box__content--lesspadding propose js-cfp-status {% if not project.cfp_state.OPEN -%} mui--hide {%- endif %}">
+    <a class="mui-btn mui-btn--accent mui-btn--raised" href="{{ project.url_for('new_proposal') }}" data-cy="propose-a-session">{% trans %}Make a submission{% endtrans %}</a>
+    {% if project.cfp_end_at_localized -%}<p class="propose__txt mui--text-light">{% trans date=project.cfp_end_at_localized|datetime(format='dd MMM YYYY, hh:mm a') %}Accepting submissions till {{ date }}{% endtrans %}</p>{% endif %}
+  </div>
+  {%- if project.cfp_state.TOGGLEABLE and not project.cfp_state.DRAFT  %}
+    <div class="project-details__box__content--lesspadding propose js-cfp-status {% if project.cfp_state.OPEN -%} mui--hide {%- endif %}">
+      <a class="mui-btn mui-btn--accent mui-btn--raised mui-btn--strike" data-cy="Make a submission">{% trans %}Make a submission{% endtrans %}</a>
+      <p class="mui--text-light mui--text-caption zero-bottom-margin">{% trans %}Submissions are closed for this project{% endtrans %}</p>
+    </div>
+  {%- endif %}
```

Comment:
What's happening in this block? Checking two states should not be done in the template. We can declare a group state instead.